src/main/java/com/epam/ta/reportportal/ws/controller/LaunchAsyncController.java

```diff
@@ -77,11 +77,10 @@ public StartLaunchRS startLaunch(@PathVariable String projectName,
 	public FinishLaunchRS finishLaunch(@PathVariable String projectName, @PathVariable String launchId,
 			@RequestBody @Validated FinishExecutionRQ finishLaunchRQ, @AuthenticationPrincipal ReportPortalUser user,
 			HttpServletRequest request) {
+		finishLaunchRQ.setBaseURL(composeBaseUrl(request.getScheme(), request.getHeader("host")));
```

Comment:
i don't think that it is a good idea to put it into rq
you should provide it as a parameter

Reply:
fixed